Dockerfile

```diff
@@ -30,19 +26,26 @@ COPY environment.yml /tmp/environment.yml
 RUN mamba env create --file /tmp/environment.yml
 
 # Set work directory for xcube installation
-RUN mkdir /xcube
-WORKDIR /xcube
+WORKDIR /home/xcube
 
 # Copy sources into xcube
-COPY . /xcube
+COPY . ./
 
 # Setup xcube package.
 RUN source activate xcube && python setup.py install
 
+WORKDIR /tmp
+ADD scripts/install_xcube-datastore.sh ./
+
+ENV XCUBE_SH_VERSION=0.8.0
```

Comment:
Can we pls move all version specifications together and top-most?

Reply:
Yes we can. done